server/api.js

```diff
@@ -102,4 +102,27 @@ router.post(
 	}
 );
 
+
+
+
+
+router.get("/search", async (req, res) => {
+	try {
+ const searchQuery = req.query.searchQuery; // Get the search query from the request query parameters
+console.log(searchQuery)
+	  // Perform the filter logic based on the search query
+ let filteredImages = await pool.query(
+   `SELECT * FROM images WHERE lower(description) LIKE $1 OR lower(tags) LIKE $1 ORDER BY rating DESC;`,
```

Comment:
I think this query will work well. It will match a lot at first but this is a good thing at the beginning when there is less content